src/docs/sanapi/available-metrics.md

```diff
@@ -83,7 +65,26 @@ Example:
 
 If you change the `aggregation` parameter to `AVG` it will return the average transaction volume over each 7 day interval. You can see how this API can be quite powerful and flexible.
 
-#### Available metrics per project
+## Available projects per metric
+
+The `getMetric` API supports a large list of assets identified by their slug. For a given metrics
+you can fetch the list of available slugs with this query:
+
+```js
+{
+  getMetric(metric: "mvrv_usd"){
+    metadata{
+      availableSlugs
+    }
+  }
+}
+```
+
+**[Run in explorer](<https://api.santiment.net/graphiql?query=%7B%0A%20%20getMetric(metric%3A%20%22mvrv_usd%22)%7B%0A%20%20%20%20metadata%7B%0A%20%20%20%20%20%20availableSlugs%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D>)**
+
+There are 2 fields that can be queried on `getMetric` and each of them has additional subfields: `metadata` and `timeseriesData`
```

Comment:
This is no longer true. The fields are now 5:
- metadata
- timeseriesData
- histogramData
- availableSince
- lastDatetimeComputedAt

I'll push some changes to this PR